Pokemon Tabletop Adventure/View Controllers/PokedexViewController.swift

```diff
@@ -24,6 +24,10 @@ class PokedexViewController: BaseViewController {
 
         self.credits.attributedText = (credits ?? PokedexViewController.iconCredits).html
         super.init(with: viewModel)
+
+        if traitCollection.forceTouchCapability == .available {
+            // TODO: Find out how we can register _each cell_ for 3D Touch!
```

Comment:
@d9rad 
Be interested to hear how you'd think is best to manage this:
We need to register something as ready for previewing here, and we need to be able to interpret which pokémon was selected at that time - we could just register the whole table view, which would be much easier, and compute which cell was tapped, but... opinions?

Reply:
Resolved the issue with closures - see next commit ;)